### PR TITLE
Add response lambda to stack

### PR DESCRIPTION
**Pull request by martinpham97**
COM-44

# Notes
- Add lambda to response to attacks (as noted in IN-AUR-01)

# Testing
- Manual tested
- TODO: integration testing/unit testing

### CODE REVIEW

File: ir_cdk_stacks/in_aur_01/response_lambda.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        if "FATAL:  password authentication failed for user" in message:
+            tok = message.split(" ")[2]
+            source = tok.split(":")[1]
+            sources.append(source)
+    
+    # Log to cloudwatch        
+    print(sources)
+    
+    # Send data to slack channel
+    webhook_url = ''
```
Comment (martinpham97) [inline ir_cdk_stacks/in_aur_01/response_lambda.py:33]: Need to find a way to input the webhook URL, maybe CLI?

File: ir_cdk_stacks/in_aur_01_stack.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    aws_iam as iam,
+)
+import os
+
+
+class InAur01Stack(core.Stack):
+
+    METRIC_NAMESPACE = "LogMetrics"
+    METRIC_NAME = "InAur01DetectionFailedDbLoginAttempts"
+    NOTIFY_EMAIL = ""
```
Comment (martinpham97) [inline ir_cdk_stacks/in_aur_01_stack.py:19]: Need to find a way to input the notification email, maybe CLI?

File: ir_cdk_stacks/in_aur_01/response_lambda.py (hunk 1 above)
Comment (TwelfthGhast) [reply ir_cdk_stacks/in_aur_01/response_lambda.py:33]: Should be ok to hardcode for now, unless you want a config file, where you can specify a common webhook url for slack and email across all IRs?

File: app.py [hunk 3]
```diff
@@ -3,11 +3,12 @@
 from aws_cdk import core
 
 from ir_cdk_stacks.ir_cdk_stacks_stack import IrCdkStacksStack
-from ir_cdk_stacks.in_aur_01_detection_stack import InAur01DetectionStack
+from ir_cdk_stacks.in_aur_01_stack import InAur01Stack
 
+env_US = core.Environment(account="544820149332", region="us-east-1")
```
Comment (TwelfthGhast) [inline app.py:8]: Will also need to generalize this and allow CLI to change this value.

File: ir_cdk_stacks/in_aur_01/response_lambda.py (hunk 1 above)
Comment (martinpham97) [reply ir_cdk_stacks/in_aur_01/response_lambda.py:33]: Slack will actually scan the web for the webhook URL. Got an email from Slack saying one of my commits has the URL and they automatically invalidate the URL.

**Review by TwelfthGhast — APPROVED** [review]

**Pull request merged**